### DISCUSSION TITLE
Ray does not support Python 3.9?

### DISCUSSION THREAD

**Discussion by RCF42** [MPoL-dev/MPoL#73] [General]
I went to go install ray just to run through and see the examples added for the 'production run' but I got this error on both Windows and Linux (both with Python 3.9):

`ERROR: Could not find a version that satisfies the requirement ray`

On their site they also don't have any Daily Releases for Python 3,9, just 3.6 through 3.8, so I think it's not a client-side problem with me. Can other people with python 3.9 confirm this?

**Comment by RCF42** [comment]
Well it seems that they have added a daily release for python 3.9 now [here ](https://docs.ray.io/en/master/installation.html)(still not on pypi though)

**Reply by iancze** [reply]
Glad that it's finally available. I'm surprised that they would be behind.
